packages/lib-utils/src/app/api-discovery/index.ts

```diff
@@ -42,6 +43,7 @@ const defineModels = (list: APIResourceList): K8sModelCommon[] => {
       ({ name, singularName, namespaced, kind, verbs, shortNames }) =>
         <K8sModelCommon>{
           ...(apiGroup ? { apiGroup } : {}),
+          name,
```

Comment:
Why do we need `name` here? Since this is the model and not a specific resource based on that model. I could have understood this wrong.

Reply:
Ah good point!